ovs_gtp/README.md

```diff
@@ -0,0 +1,45 @@
+## Cleanup ovs build with Ansible.
+
+Goal: Remove the dependency on other Git repos by moving the build process for ovs to ansible and checkin the associated patches into a locally maintained subdir under magma/ovs_gtp
+
+In `roles/ovs_gtp`, you can find the main task in order to build the associate patches.
+
+#### Requirement
+---
+
+```
+ansible <= 2
+python
+```
+
+#### How to launch a build
+---
+
+First of all fill the hosts file, you can either do
+
+- Local build
+
+```[local]
+127.0.0.1   ansible_connection=local
+```
+- Vagrant
+
+```[localvagrant]
+127.0.0.1   ansible_ssh_user=vagrant    ansible_ssh_port=2222   ansible_ssh_private_key_file=~/.vagrant.d/insecure_private_key
```

Comment:
Can we use the hosts file in https://github.com/facebookincubator/magma/blob/master/lte/gateway/deploy/hosts? Looks like we can simplify the usage more :).